imports/client/components/Puzzle.tsx

```diff
@@ -64,7 +64,7 @@ class Puzzle extends React.PureComponent<PuzzleProps, PuzzleState> {
   editButton = () => {
     if (this.props.canUpdate) {
       return (
-        <Button onClick={this.showEditModal} bsStyle="default" bsSize="xs" title="Edit puzzle...">
+        <Button onClick={this.showEditModal} variant="secondary" size="sm" title="Edit puzzle...">
```

Comment:
this went from bsSize=xs to size=sm -- intentional?

Reply:
Yeah, buttons [no longer have an "xs" size](https://getbootstrap.com/docs/4.5/migration/#:~:text=Dropped%20the%20.btn-xs%20class%20entirely).